### PR TITLE
wrote my review on debian

### PR DESCRIPTION
**Pull request by reonjames**
hope u like my review

### CODE REVIEW

File: kubuntu.md [hunk 1]
```diff
@@ -1,31 +1,32 @@
-# Kubuntu
+# DISTRO NAME
```
Comment (subins2000) [inline kubuntu.md]: Why remove everything ??

**Comment by subins2000** [comment]
@reonjames Don't change Kubuntu file. It's already written well by @aishwar-yaaa . Please don't undo her work !

**Comment by reonjames** [comment]
Ok..it's my fault srry @subin2000..

**Comment by reonjames** [comment]
ok..srry its my fault @subins2000

**Comment by subins2000** [comment]
No problemo ! It's your first time and you're already doing really well !

Way to fix it is :
* Revert `kubuntu.md` file back to what it was : `git checkout previous_commit_id -- kubuntu.md`
* Amend your commit : `git commit debian.md --amend`
* And force a push : `git push -f`

That's it !

**Pull request merged**